### PR TITLE
#231 Invoking kompose --bundle X.dab convert --stdout will produce tw…

### PR DESCRIPTION
**Pull request by cab105**
…o differently ordered results

### CODE REVIEW

File: pkg/transformer/kubernetes/kubernetes.go [hunk 1]
```diff
@@ -332,7 +333,15 @@ func (k *Kubernetes) Transform(komposeObject kobject.KomposeObject, opt kobject.
 	// this will hold all the converted data
 	var allobjects []runtime.Object
 
-	for name, service := range komposeObject.ServiceConfigs {
+	// Need to ensure that the kerbenertes objects are in a consistent order
```
Comment (cdrage) [inline pkg/transformer/kubernetes/kubernetes.go]: erh meh gerd, kerbenertes! just a small spelling error :)

**Comment by cab105** [comment]
Whoopsie.  Knew I needed another cup of coffee before pushing.

**Comment by cab105** [comment]
Normally yes, however this bug was the result of adding in unit tests for dab/dsb support (See #241)

**Comment by cdrage** [comment]
@cab105 ah k, merge away then? :+1:

**Comment by cab105** [comment]
@cdrage.  Unfortunately blocked.  Will need @ngtuna to bless this.

**Comment by ngtuna** [comment]
@cab105 Done. Go ahead man :-)

**Comment by cab105** [comment]
Thanks @ngtuna!

**Pull request merged**